tests/113-test-container-root-volume-cleanup.sh

```diff
@@ -1,20 +1,20 @@
 source $SRCDIR/libtest.sh
 
-# Test "docker-storage-setup --reset" for CONTAINER_ROOT_LV_NAME=
```

Comment:
Can we keep all the changes related to tests either in a separate PR or in a separate patch. I wan to make sure core changes are part of a separate commit.